### ISSUE TITLE
webapp:Get-AzWebAppCertificate does not return any results

### ISSUE THREAD

**Issue by RPCMarkR** [Azure/azure-powershell#11110]
## Description
When running 
$Certificates = Get-AzWebAppCertificate -ResourceGroupName "mySampleResourceGroupName" 

$Certificates is blank. I cannot get any properties, members or anything from the variable
"$Certificates -eq $null" returns nothing, neither true nor false.
Previously, I was able to loop through the $Certificates to find the certificate name I wanted and then get the thumbprint.

I can successfully bind an existing certificate using the command below, (if I manually set the $thumbprint) therefore I know the certificate is there.

New-AzWebAppSSLBinding `
 -WebAppName $WebAppName `
 -ResourceGroupName "mySampleResourceGroup" `
 -Name "myapp.mydomain.com" `
 -Thumbprint "$thumbprint" `
 -SslState SniEnabled 

**NOTE: The ASP is in a different resource group than the Web App Service.

## Steps to reproduce

```powershell
$asp = Get-AzAppServicePlan -ResourceGroup "myASPResourceGroupName" -Name "mySampleASP"
$WebApp = New-AzWebApp -ResourceGroupName "mySampleResourceGroupName" -Name "$WebAppName" -Location "North Central US" -AppServicePlan $asp.Id 

New-AzWebAppSSLBinding `
 -WebAppName $WebAppName `
 -ResourceGroupName "mySampleResourceGroupName" `
 -Name "myapp.mydomain.com" `
 -CertificateFilePath "$Script:WebCertificate" `
 -CertificatePassword "$PlainTextPassword" `
 -SslState SniEnabled `

$Certificates = Get-AzWebAppCertificate -ResourceGroupName "mySampleResourceGroupName"
#$Certificates is empty
            if ($Certificates -ne $null) { #does not work
                foreach ($Certificate in $Certificates) {
                    if ($Certificate.FriendlyName -eq "sampledomain.com") {
                        $FoundCWCertificate = $true
                        $thumbprint = $Certificate.Thumbprint
                    }
                }
            } 
            if ($FoundCWCertificate) {
                New-AzWebAppSSLBinding `
                    -WebAppName $WebAppName `
                    -ResourceGroupName "mySampleResourceGroupName)" `
                    -Name "myapp.mydomain.com" `
                    -Thumbprint "$thumbprint" `
                    -SslState SniEnabled `
                    -ErrorAction Stop
            }
            else {
                New-AzWebAppSSLBinding `
                    -WebAppName $WebAppName `
                    -ResourceGroupName "mySampleResourceGroupName" `
                    -Name "myapp.mydomain.com" `
                    -CertificateFilePath "$Script:WebCertificate" `
                    -CertificatePassword "$PlainTextPassword" `
                    -SslState SniEnabled `
                    -ErrorAction Stop
            }

```

## Environment data

```
Name                           Value
----                           -----
PSVersion                      5.1.18362.145
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.18362.145
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1

```

## Module versions

```powershell
ModuleType Version    Name                                ExportedCommands
---------- -------    ----                                ----------------
Script     1.7.1      Az.Accounts                         {Disable-AzDataCollection, Disable-AzContextAutosave, Enab...
Script     1.1.1      Az.Advisor                          {Get-AzAdvisorRecommendation, Enable-AzAdvisorRecommendati...
Script     1.0.3      Az.Aks                              {Get-AzAks, New-AzAks, Remove-AzAks, Import-AzAksCredentia...
Script     1.1.2      Az.AnalysisServices                 {Resume-AzAnalysisServicesServer, Suspend-AzAnalysisServic...
Script     1.3.4      Az.ApiManagement                    {Add-AzApiManagementApiToProduct, Add-AzApiManagementProdu...
Script     1.0.3      Az.ApplicationInsights              {Get-AzApplicationInsights, New-AzApplicationInsights, Rem...
Script     1.3.5      Az.Automation                       {Get-AzAutomationHybridWorkerGroup, Remove-AzAutomationHyb...
Script     2.0.2      Az.Batch                            {Remove-AzBatchAccount, Get-AzBatchAccount, Get-AzBatchAcc...
Script     1.0.2      Az.Billing                          {Get-AzBillingInvoice, Get-AzBillingPeriod, Get-AzEnrollme...
Script     1.4.2      Az.Cdn                              {Get-AzCdnProfile, Get-AzCdnProfileSsoUrl, New-AzCdnProfil...
Script     1.2.2      Az.CognitiveServices                {Get-AzCognitiveServicesAccount, Get-AzCognitiveServicesAc...
Script     3.4.0      Az.Compute                          {Remove-AzAvailabilitySet, Get-AzAvailabilitySet, New-AzAv...
Script     1.0.3      Az.ContainerInstance                {New-AzContainerGroup, Get-AzContainerGroup, Remove-AzCont...
Script     1.1.1      Az.ContainerRegistry                {New-AzContainerRegistry, Get-AzContainerRegistry, Update-...
Script     1.1.0      Az.DataBoxEdge                      {Get-AzDataBoxEdgeJob, Get-AzDataBoxEdgeDevice, Invoke-AzD...
Script     1.6.1      Az.DataFactory                      {Set-AzDataFactoryV2, Update-AzDataFactoryV2, Get-AzDataFa...
Script     1.0.2      Az.DataLakeAnalytics                {Get-AzDataLakeAnalyticsDataSource, New-AzDataLakeAnalytic...
Script     1.2.6      Az.DataLakeStore                    {Get-AzDataLakeStoreTrustedIdProvider, Remove-AzDataLakeSt...
Script     1.1.0      Az.DeploymentManager                {Get-AzDeploymentManagerArtifactSource, New-AzDeploymentMa...
Script     1.0.2      Az.DevTestLabs                      {Get-AzDtlAllowedVMSizesPolicy, Get-AzDtlAutoShutdownPolic...
Script     1.1.2      Az.Dns                              {Get-AzDnsRecordSet, New-AzDnsRecordConfig, Remove-AzDnsRe...
Script     1.2.3      Az.EventGrid                        {New-AzEventGridTopic, Get-AzEventGridTopic, Set-AzEventGr...
Script     1.4.3      Az.EventHub                         {New-AzEventHubNamespace, Get-AzEventHubNamespace, Set-AzE...
Script     1.3.0      Az.FrontDoor                        {New-AzFrontDoor, Get-AzFrontDoor, Set-AzFrontDoor, Remove...
Script     3.0.3      Az.HDInsight                        {Get-AzHDInsightJob, New-AzHDInsightSqoopJobDefinition, Wa...
Script     1.0.1      Az.HealthcareApis                   {New-AzHealthcareApisService, Remove-AzHealthcareApisServi...
Script     2.0.1      Az.IotHub                           {Add-AzIotHubKey, Get-AzIotHubEventHubConsumerGroup, Get-A...
Script     1.5.0      Az.KeyVault                         {Add-AzKeyVaultCertificate, Update-AzKeyVaultCertificate, ...
Script     1.3.2      Az.LogicApp                         {Get-AzIntegrationAccountAgreement, Get-AzIntegrationAccou...
Script     1.1.3      Az.MachineLearning                  {Move-AzMlCommitmentAssociation, Get-AzMlCommitmentAssocia...
Script     1.0.2      Az.ManagedServices                  {Get-AzManagedServicesAssignment, New-AzManagedServicesAss...
Script     1.0.2      Az.MarketplaceOrdering              {Get-AzMarketplaceTerms, Set-AzMarketplaceTerms}
Script     1.1.1      Az.Media                            {Sync-AzMediaServiceStorageKey, Set-AzMediaServiceKey, Get...
Script     1.5.0      Az.Monitor                          {Get-AzMetricDefinition, Get-AzMetric, Remove-AzLogProfile...
Script     2.3.0      Az.Network                          {Add-AzApplicationGatewayAuthenticationCertificate, Get-Az...
Script     1.1.1      Az.NotificationHubs                 {Get-AzNotificationHub, Get-AzNotificationHubAuthorization...
Script     1.3.4      Az.OperationalInsights              {New-AzOperationalInsightsAzureActivityLogDataSource, New-...
Script     1.2.0      Az.PolicyInsights                   {Get-AzPolicyEvent, Get-AzPolicyState, Get-AzPolicyStateSu...
Script     1.1.1      Az.PowerBIEmbedded                  {Remove-AzPowerBIWorkspaceCollection, Get-AzPowerBIWorkspa...
Script     1.0.2      Az.PrivateDns                       {Get-AzPrivateDnsZone, Remove-AzPrivateDnsZone, Set-AzPriv...
Script     2.5.0      Az.RecoveryServices                 {Get-AzRecoveryServicesBackupProperty, Get-AzRecoveryServi...
Script     1.2.1      Az.RedisCache                       {Remove-AzRedisCachePatchSchedule, New-AzRedisCacheSchedul...
Script     1.0.3      Az.Relay                            {New-AzRelayNamespace, Get-AzRelayNamespace, Set-AzRelayNa...
Script     1.10.0     Az.Resources                        {Get-AzProviderOperation, Remove-AzRoleAssignment, Get-AzR...
Script     1.4.1      Az.ServiceBus                       {New-AzServiceBusNamespace, Get-AzServiceBusNamespace, Set...
Script     2.0.1      Az.ServiceFabric                    {Add-AzServiceFabricClientCertificate, Add-AzServiceFabric...
Script     1.1.1      Az.SignalR                          {New-AzSignalR, Get-AzSignalR, Get-AzSignalRKey, New-AzSig...
Script     2.2.0      Az.Sql                              {Get-AzSqlDatabaseTransparentDataEncryption, Get-AzSqlData...
Script     1.0.2      Az.SqlVirtualMachine                {New-AzSqlVM, Get-AzSqlVM, Update-AzSqlVM, Remove-AzSqlVM...}
Script     1.12.0     Az.Storage                          {Get-AzStorageAccount, Get-AzStorageAccountKey, New-AzStor...
Script     1.2.2      Az.StorageSync                      {Invoke-AzStorageSyncCompatibilityCheck, New-AzStorageSync...
Script     1.0.1      Az.StreamAnalytics                  {Get-AzStreamAnalyticsFunction, Get-AzStreamAnalyticsDefau...
Script     1.0.3      Az.TrafficManager                   {Add-AzTrafficManagerCustomHeaderToEndpoint, Remove-AzTraf...
Script     1.6.0      Az.Websites                         {Get-AzAppServicePlan, Set-AzAppServicePlan, New-AzAppServ...
Script     5.1.2      Azure                               {Get-AzureAutomationCertificate, Get-AzureAutomationConnec...
Script     5.7.0      AzureRM
Script     0.6.6      AzureRM.AnalysisServices            {Resume-AzureRmAnalysisServicesServer, Suspend-AzureRmAnal...
Script     5.1.2      AzureRM.ApiManagement               {Add-AzureRmApiManagementRegion, Get-AzureRmApiManagementS...
Script     0.1.3      AzureRM.ApplicationInsights         {Get-AzureRmApplicationInsights, New-AzureRmApplicationIns...
Script     4.3.2      AzureRM.Automation                  {Get-AzureRMAutomationHybridWorkerGroup, Get-AzureRmAutoma...
Script     4.0.4      AzureRM.Backup                      {Backup-AzureRmBackupItem, Enable-AzureRmBackupContainerRe...
Script     4.0.6      AzureRM.Batch                       {Remove-AzureRmBatchAccount, Get-AzureRmBatchAccount, Get-...
Script     0.14.1     AzureRM.Billing                     {Get-AzureRmBillingInvoice, Get-AzureRmBillingPeriod, Get-...
Script     4.2.2      AzureRM.Cdn                         {Get-AzureRmCdnProfile, Get-AzureRmCdnProfileSsoUrl, New-A...
Script     0.9.4      AzureRM.CognitiveServices           {Get-AzureRmCognitiveServicesAccount, Get-AzureRmCognitive...
Script     4.6.0      AzureRM.Compute                     {Remove-AzureRmAvailabilitySet, Get-AzureRmAvailabilitySet...
Script     0.3.1      AzureRM.Consumption                 Get-AzureRmConsumptionUsageDetail
Script     0.2.5      AzureRM.ContainerInstance           {New-AzureRmContainerGroup, Get-AzureRmContainerGroup, Rem...
Script     1.0.4      AzureRM.ContainerRegistry           {New-AzureRmContainerRegistry, Get-AzureRmContainerRegistr...
Script     4.2.2      AzureRM.DataFactories               {Remove-AzureRmDataFactory, Get-AzureRmDataFactoryRun, Get...
Script     0.5.3      AzureRM.DataFactoryV2               {Set-AzureRmDataFactoryV2, Update-AzureRmDataFactoryV2, Ge...
Script     4.2.3      AzureRM.DataLakeAnalytics           {Get-AzureRmDataLakeAnalyticsDataSource, New-AzureRmDataLa...
Script     5.2.0      AzureRM.DataLakeStore               {Get-AzureRmDataLakeStoreTrustedIdProvider, Remove-AzureRm...
Script     4.0.4      AzureRM.DevTestLabs                 {Get-AzureRmDtlAllowedVMSizesPolicy, Get-AzureRmDtlAutoShu...
Script     4.1.2      AzureRM.Dns                         {Get-AzureRmDnsRecordSet, New-AzureRmDnsRecordConfig, Remo...
Script     0.3.2      AzureRM.EventGrid                   {New-AzureRmEventGridTopic, Get-AzureRmEventGridTopic, Set...
Script     0.6.3      AzureRM.EventHub                    {New-AzureRmEventHubNamespace, Get-AzureRmEventHubNamespac...
Script     4.1.2      AzureRM.HDInsight                   {Get-AzureRmHDInsightJob, New-AzureRmHDInsightSqoopJobDefi...
Script     4.0.4      AzureRM.Insights                    {Get-AzureRmMetricDefinition, Get-AzureRmMetric, Remove-Az...
Script     3.1.2      AzureRM.IotHub                      {Add-AzureRmIotHubKey, Get-AzureRmIotHubEventHubConsumerGr...
Script     4.3.0      AzureRM.KeyVault                    {Add-AzureKeyVaultCertificate, Set-AzureKeyVaultCertificat...
Script     4.0.3      AzureRM.LogicApp                    {Get-AzureRmIntegrationAccountAgreement, Get-AzureRmIntegr...
Script     0.17.2     AzureRM.MachineLearning             {Move-AzureRmMlCommitmentAssociation, Get-AzureRmMlCommitm...
Script     0.4.2      AzureRM.MachineLearningCompute      {Get-AzureRmMlOpCluster, Get-AzureRmMlOpClusterKey, Test-A...
Script     0.2.1      AzureRM.MarketplaceOrdering         {Get-AzureRmMarketplaceTerms, Set-AzureRmMarketplaceTerms}
Script     0.9.2      AzureRM.Media                       {Sync-AzureRmMediaServiceStorageKeys, Set-AzureRmMediaServ...
Script     5.4.2      AzureRM.Network                     {Add-AzureRmApplicationGatewayAuthenticationCertificate, G...
Script     4.1.1      AzureRM.NotificationHubs            {Get-AzureRmNotificationHub, Get-AzureRmNotificationHubAut...
Script     4.3.2      AzureRM.OperationalInsights         {New-AzureRmOperationalInsightsAzureActivityLogDataSource,...
Script     4.1.4      AzureRM.PowerBIEmbedded             {Remove-AzureRmPowerBIWorkspaceCollection, Get-AzureRmPowe...
Script     4.6.0      AzureRM.Profile                     {Disable-AzureRmDataCollection, Disable-AzureRmContextAuto...
Script     4.1.2      AzureRM.RecoveryServices            {Get-AzureRmRecoveryServicesBackupProperty, Get-AzureRmRec...
Script     4.1.2      AzureRM.RecoveryServices.Backup     {Backup-AzureRmRecoveryServicesBackupItem, Get-AzureRmReco...
Script     0.2.4      AzureRM.RecoveryServices.SiteRec... {Edit-AzureRmRecoveryServicesAsrRecoveryPlan, Get-AzureRmR...
Script     4.1.2      AzureRM.RedisCache                  {Remove-AzureRmRedisCachePatchSchedule, New-AzureRmRedisCa...
Script     0.3.3      AzureRM.Relay                       {New-AzureRmRelayNamespace, Get-AzureRmRelayNamespace, Set...
Script     5.5.2      AzureRM.Resources                   {Get-AzureRmProviderOperation, Remove-AzureRmRoleAssignmen...
Script     0.16.3     AzureRM.Scheduler                   {Disable-AzureRmSchedulerJobCollection, Enable-AzureRmSche...
Script     4.1.2      AzureRM.ServerManagement            {Invoke-AzureRmServerManagementPowerShellCommand, Get-Azur...
Script     0.6.4      AzureRM.ServiceBus                  {New-AzureRmServiceBusNamespace, Get-AzureRmServiceBusName...
Script     0.3.4      AzureRM.ServiceFabric               {Add-AzureRmServiceFabricApplicationCertificate, Add-Azure...
Script     5.0.6      AzureRM.SiteRecovery                {Get-AzureRmSiteRecoveryFabric, New-AzureRmSiteRecoveryFab...
Script     4.4.0      AzureRM.Sql                         {Get-AzureRmSqlDatabaseTransparentDataEncryption, Get-Azur...
Script     4.2.3      AzureRM.Storage                     {Get-AzureRmStorageAccount, Get-AzureRmStorageAccountKey, ...
Script     4.0.4      AzureRM.StreamAnalytics             {Get-AzureRmStreamAnalyticsFunction, Get-AzureRmStreamAnal...
Script     4.0.1      AzureRM.Tags                        {Remove-AzureRmTag, Get-AzureRmTag, New-AzureRmTag}
Script     4.0.3      AzureRM.TrafficManager              {Disable-AzureRmTrafficManagerEndpoint, Enable-AzureRmTraf...
Script     4.0.2      AzureRM.UsageAggregates             Get-UsageAggregates
Script     4.2.2      AzureRM.Websites                    {Get-AzureRmAppServicePlan, Set-AzureRmAppServicePlan, New...
Script     1.0.1      Microsoft.PowerShell.Operation.V... {Get-OperationValidation, Invoke-OperationValidation}
Binary     1.0.0.1    PackageManagement                   {Find-Package, Get-Package, Get-PackageProvider, Get-Packa...
Script     3.4.0      Pester                              {Describe, Context, It, Should...}
Script     1.0.0.1    PowerShellGet                       {Install-Module, Find-Module, Save-Module, Update-Module...}
Script     2.0.0      PSReadline                          {Get-PSReadLineKeyHandler, Set-PSReadLineKeyHandler, Remov...
Script     21.0.17279 SqlServer                           {Add-RoleMember, Add-SqlAvailabilityDatabase, Add-SqlAvail...


    Directory: C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules


ModuleType Version    Name                                ExportedCommands
---------- -------    ----                                ----------------
Manifest   1.0.1.0    ActiveDirectory                     {Add-ADCentralAccessPolicyMember, Add-ADComputerServiceAcc...
Manifest   1.0.0.0    AppBackgroundTask                   {Disable-AppBackgroundTaskDiagnosticLog, Enable-AppBackgro...
Manifest   2.0.0.0    AppLocker                           {Get-AppLockerFileInformation, Get-AppLockerPolicy, New-Ap...
Manifest   1.0.0.0    AppvClient                          {Add-AppvClientConnectionGroup, Add-AppvClientPackage, Add...
Manifest   2.0.1.0    Appx                                {Add-AppxPackage, Get-AppxPackage, Get-AppxPackageManifest...
Script     1.0.0.0    AssignedAccess                      {Clear-AssignedAccess, Get-AssignedAccess, Set-AssignedAcc...
Manifest   1.0        BestPractices                       {Get-BpaModel, Get-BpaResult, Invoke-BpaModel, Set-BpaResult}
Manifest   1.0.0.0    BitLocker                           {Unlock-BitLocker, Suspend-BitLocker, Resume-BitLocker, Re...
Manifest   2.0.0.0    BitsTransfer                        {Add-BitsFile, Complete-BitsTransfer, Get-BitsTransfer, Re...
Manifest   1.0.0.0    BranchCache                         {Add-BCDataCacheExtension, Clear-BCCache, Disable-BC, Disa...
Manifest   1.0.0.0    CimCmdlets                          {Get-CimAssociatedInstance, Get-CimClass, Get-CimInstance,...
Binary     2.0.0.0    ClusterAwareUpdating                {Get-CauPlugin, Register-CauPlugin, Unregister-CauPlugin, ...
Manifest   1.0        ConfigCI                            {Get-SystemDriver, New-CIPolicyRule, New-CIPolicy, Get-CIP...
Manifest   1.0        ConfigDefender                      {Get-MpPreference, Set-MpPreference, Add-MpPreference, Rem...
Manifest   1.0        Defender                            {Get-MpPreference, Set-MpPreference, Add-MpPreference, Rem...
Manifest   1.0.2.0    DeliveryOptimization                {Delete-DeliveryOptimizationCache, Get-DeliveryOptimizatio...
Manifest   1.0        DFSN                                {Get-DfsnRoot, Remove-DfsnRoot, Set-DfsnRoot, New-DfsnRoot...
Binary     2.0.0.0    DFSR                                {New-DfsReplicationGroup, Get-DfsReplicationGroup, Set-Dfs...
Manifest   2.0.0.0    DhcpServer                          {Add-DhcpServerInDC, Add-DhcpServerv4Class, Add-DhcpServer...
Manifest   1.0.0.0    DirectAccessClientComponents        {Disable-DAManualEntryPointSelection, Enable-DAManualEntry...
Script     3.0        Dism                                {Add-AppxProvisionedPackage, Add-WindowsDriver, Add-Window...
Manifest   1.0.0.0    DnsClient                           {Resolve-DnsName, Clear-DnsClientCache, Get-DnsClient, Get...
Manifest   2.0.0.0    DnsServer                           {Add-DnsServerConditionalForwarderZone, Add-DnsServerDirec...
Manifest   1.0.0.0    EventTracingManagement              {Start-EtwTraceSession, New-EtwTraceSession, Get-EtwTraceS...
Manifest   2.0.0.0    FailoverClusters                    {Add-ClusterCheckpoint, Add-ClusterDisk, Add-ClusterFileSe...
Manifest   1.0.0.0    GroupPolicy                         {Backup-GPO, Block-GPInheritance, Copy-GPO, Get-GPInherita...
Manifest   1.0.0.0    HgsClient                           {Get-HgsAttestationBaselinePolicy, Get-HgsClientConfigurat...
Manifest   1.0.0.0    HgsDiagnostics                      {New-HgsTraceTarget, Get-HgsTrace, Get-HgsTraceFileData, T...
Manifest   1.0.0.1    HostNetworkingService               {Remove-HnsNamespace, Remove-HnsEndpoint, Get-HnsEndpoint,...
Binary     2.0.0.0    Hyper-V                             {Add-VMAssignableDevice, Add-VMDvdDrive, Add-VMFibreChanne...
Binary     1.1        Hyper-V                             {Add-VMDvdDrive, Add-VMFibreChannelHba, Add-VMHardDiskDriv...
Script     1.1.0.0    IISAdministration                   {Get-IISAppPool, Start-IISCommitDelay, Stop-IISCommitDelay...
Manifest   2.0.0.0    International                       {Get-WinDefaultInputMethodOverride, Set-WinDefaultInputMet...
Manifest   2.0.0.0    IpamServer                          {Get-IpamDhcpConfigurationEvent, Remove-IpamDhcpConfigurat...
Manifest   1.0.0.0    iSCSI                               {Get-IscsiTargetPortal, New-IscsiTargetPortal, Remove-Iscs...
Manifest   2.0.0.0    IscsiTarget                         {Add-ClusteriSCSITargetServerRole, Add-IscsiVirtualDiskTar...
Script     1.0.0.0    ISE                                 {New-IseSnippet, Import-IseSnippet, Get-IseSnippet}
Manifest   1.0.0.0    Kds                                 {Add-KdsRootKey, Get-KdsRootKey, Test-KdsRootKey, Set-KdsC...
Manifest   1.0.1.0    Microsoft.PowerShell.Archive        {Compress-Archive, Expand-Archive}
Manifest   3.0.0.0    Microsoft.PowerShell.Diagnostics    {Get-WinEvent, Get-Counter, Import-Counter, Export-Counter...
Manifest   3.0.0.0    Microsoft.PowerShell.Host           {Start-Transcript, Stop-Transcript}
Manifest   1.0.0.0    Microsoft.PowerShell.LocalAccounts  {Add-LocalGroupMember, Disable-LocalUser, Enable-LocalUser...
Manifest   3.1.0.0    Microsoft.PowerShell.Management     {Add-Content, Clear-Content, Clear-ItemProperty, Join-Path...
Script     1.0        Microsoft.PowerShell.ODataUtils     Export-ODataEndpointProxy
Manifest   3.0.0.0    Microsoft.PowerShell.Security       {Get-Acl, Set-Acl, Get-PfxCertificate, Get-Credential...}
Manifest   3.1.0.0    Microsoft.PowerShell.Utility        {Format-List, Format-Custom, Format-Table, Format-Wide...}
Manifest   3.0.0.0    Microsoft.WSMan.Management          {Disable-WSManCredSSP, Enable-WSManCredSSP, Get-WSManCredS...
Manifest   1.0        MMAgent                             {Disable-MMAgent, Enable-MMAgent, Set-MMAgent, Get-MMAgent...
Manifest   1.0.0.0    MsDtc                               {New-DtcDiagnosticTransaction, Complete-DtcDiagnosticTrans...
Binary     1.0.0.0    MSMQ                                {Clear-MsmqOutgoingQueue, Clear-MsmqQueue, Enable-MsmqCert...
Manifest   2.0.0.0    NetAdapter                          {Disable-NetAdapter, Disable-NetAdapterBinding, Disable-Ne...
Manifest   1.0.0.0    NetConnection                       {Get-NetConnectionProfile, Set-NetConnectionProfile}
Manifest   1.0.0.0    NetDiagnostics                      Get-NetView
Manifest   1.0.0.0    NetEventPacketCapture               {New-NetEventSession, Remove-NetEventSession, Get-NetEvent...
Manifest   2.0.0.0    NetLbfo                             {Add-NetLbfoTeamMember, Add-NetLbfoTeamNic, Get-NetLbfoTea...
Manifest   1.0        NetLldpAgent                        {Enable-NetLldpAgent, Disable-NetLldpAgent, Get-NetLldpAgent}
Manifest   1.0.0.0    NetNat                              {Get-NetNat, Get-NetNatExternalAddress, Get-NetNatStaticMa...
Manifest   2.0.0.0    NetQos                              {Get-NetQosPolicy, Set-NetQosPolicy, Remove-NetQosPolicy, ...
Manifest   2.0.0.0    NetSecurity                         {Get-DAPolicyChange, New-NetIPsecAuthProposal, New-NetIPse...
Manifest   1.0.0.0    NetSwitchTeam                       {New-NetSwitchTeam, Remove-NetSwitchTeam, Get-NetSwitchTea...
Manifest   1.0.0.0    NetTCPIP                            {Get-NetIPAddress, Get-NetIPInterface, Get-NetIPv4Protocol...
Manifest   1.0.0.0    NetworkConnectivityStatus           {Get-DAConnectionStatus, Get-NCSIPolicyConfiguration, Rese...
Manifest   1.0.0.0    NetworkController                   {Add-NetworkControllerNode, Disable-NetworkControllerNode,...
Manifest   1.0.0.0    NetworkControllerDiagnostics        {Get-NetworkControllerDeploymentInfo, Get-NetworkControlle...
Manifest   2.0.0.0    NetworkLoadBalancingClusters        {Add-NlbClusterNode, Add-NlbClusterNodeDip, Add-NlbCluster...
Manifest   1.0.0.0    NetworkSwitchManager                {Disable-NetworkSwitchEthernetPort, Enable-NetworkSwitchEt...
Manifest   1.0.0.0    NetworkTransition                   {Add-NetIPHttpsCertBinding, Disable-NetDnsTransitionConfig...
Manifest   1.0        NFS                                 {Get-NfsMappedIdentity, Get-NfsNetgroup, Install-NfsMappin...
Manifest   1.0.0.0    PcsvDevice                          {Get-PcsvDevice, Start-PcsvDevice, Stop-PcsvDevice, Restar...
Binary     1.0.0.0    PersistentMemory                    {Get-PmemDisk, Get-PmemPhysicalDevice, Get-PmemUnusedRegio...
Manifest   1.0.0.0    PKI                                 {Add-CertificateEnrollmentPolicyServer, Export-Certificate...
Manifest   1.0.0.0    PnpDevice                           {Get-PnpDevice, Get-PnpDeviceProperty, Enable-PnpDevice, D...
Manifest   1.1        PrintManagement                     {Add-Printer, Add-PrinterDriver, Add-PrinterPort, Get-Prin...
Binary     1.0.11     ProcessMitigations                  {Get-ProcessMitigation, Set-ProcessMitigation, ConvertTo-P...
Script     3.0        Provisioning                        {Install-ProvisioningPackage, Export-ProvisioningPackage, ...
Manifest   1.1        PSDesiredStateConfiguration         {Set-DscLocalConfigurationManager, Start-DscConfiguration,...
Script     1.0.0.0    PSDiagnostics                       {Disable-PSTrace, Disable-PSWSManCombinedTrace, Disable-WS...
Binary     1.1.0.0    PSScheduledJob                      {New-JobTrigger, Add-JobTrigger, Remove-JobTrigger, Get-Jo...
Manifest   2.0.0.0    PSWorkflow                          {New-PSWorkflowExecutionOption, New-PSWorkflowSession, nwsn}
Manifest   1.0.0.0    PSWorkflowUtility                   Invoke-AsWorkflow
Manifest   3.0.0.0    RemoteAccess                        {Add-DAAppServer, Add-DAClient, Add-DAClientDnsConfigurati...
Manifest   2.0.0.0    RemoteDesktop                       {Get-RDCertificate, Set-RDCertificate, New-RDCertificate, ...
Manifest   1.0.0.0    ScheduledTasks                      {Get-ScheduledTask, Set-ScheduledTask, Register-ScheduledT...
Manifest   2.0.0.0    SecureBoot                          {Confirm-SecureBootUEFI, Set-SecureBootUEFI, Get-SecureBoo...
Script     2.0.0.0    ServerManager                       {Get-WindowsFeature, Install-WindowsFeature, Uninstall-Win...
Cim        1.0.0.0    ServerManagerTasks                  {Get-SMCounterSample, Get-SMPerformanceCollector, Start-SM...
Manifest   1.0.0.0    ShieldedVMDataFile                  {Import-ShieldingDataFile, New-ShieldingDataFile, New-Volu...
Manifest   1.0.0.0    ShieldedVMTemplate                  {Protect-TemplateDisk, Initialize-VMShieldingHelperVHD}
Manifest   2.0.0.0    SmbShare                            {Get-SmbShare, Remove-SmbShare, Set-SmbShare, Block-SmbSha...
Manifest   2.0.0.0    SmbWitness                          {Get-SmbWitnessClient, Move-SmbWitnessClient, gsmbw, msmbw...
Manifest   1.0.0.0    StartLayout                         {Export-StartLayout, Import-StartLayout, Export-StartLayou...
Manifest   2.0.0.0    Storage                             {Add-InitiatorIdToMaskingSet, Add-PartitionAccessPath, Add...
Manifest   1.0.0.0    StorageBusCache                     {Clear-StorageBusDisk, Disable-StorageBusCache, Disable-St...
Binary     1.0.0.0    StorageMigrationService             {Get-SmsInventory, New-SmsInventory, Remove-SmsInventory, ...
Manifest   1.0.0.0    StorageQoS                          {Get-StorageQoSPolicy, Get-StorageQoSPolicyStore, Set-Stor...
Manifest   1.0        StorageReplica                      {Test-SRTopology, New-SRGroup, Remove-SRGroup, Set-SRGroup...
Binary     1.0.0.0    SystemInsights                      {Add-InsightsCapability, Remove-InsightsCapability, Update...
Manifest   2.0.0.0    TLS                                 {New-TlsSessionTicketKey, Enable-TlsSessionTicketKey, Disa...
Manifest   1.0.0.0    TroubleshootingPack                 {Get-TroubleshootingPack, Invoke-TroubleshootingPack}
Manifest   2.0.0.0    TrustedPlatformModule               {Get-Tpm, Initialize-Tpm, Clear-Tpm, Unblock-Tpm...}
Binary     2.1.639.0  UEV                                 {Clear-UevConfiguration, Clear-UevAppxPackage, Restore-Uev...
Manifest   2.0.0.0    UpdateServices                      {Add-WsusComputer, Approve-WsusUpdate, Deny-WsusUpdate, Ge...
Manifest   2.0.0.0    VpnClient                           {Add-VpnConnection, Set-VpnConnection, Remove-VpnConnectio...
Manifest   1.0.0.0    Wdac                                {Get-OdbcDriver, Set-OdbcDriver, Get-OdbcDsn, Add-OdbcDsn...}
Manifest   1.0.0.0    WebAdministration                   {Start-WebCommitDelay, Stop-WebCommitDelay, Get-WebConfigu...
Manifest   2.0.0.0    Whea                                {Get-WheaMemoryPolicy, Set-WheaMemoryPolicy}
Manifest   1.0.0.0    WindowsDeveloperLicense             {Get-WindowsDeveloperLicense, Unregister-WindowsDeveloperL...
Script     1.0        WindowsErrorReporting               {Enable-WindowsErrorReporting, Disable-WindowsErrorReporti...
Manifest   1.0.0.0    WindowsSearch                       {Get-WindowsSearchSetting, Set-WindowsSearchSetting}
Manifest   1.0.0.0    WindowsUpdate                       Get-WindowsUpdateLog
Manifest   1.0.0.2    WindowsUpdateProvider               {Get-WUAVersion, Get-WULastInstallationDate, Get-WULastSca...


    Directory: C:\Program Files (x86)\Microsoft SQL Server\140\Tools\PowerShell\Modules


ModuleType Version    Name                                ExportedCommands
---------- -------    ----                                ----------------
Manifest   14.0       SQLPS                               {Backup-SqlDatabase, Save-SqlMigrationReport, Add-SqlAvail...


```

## Debug output

```
nothing returned
```

## Error output

```
no errors from this
```

**Comment by wyunchi-ms** [comment]
Hi @RPCMarkR , thanks a lot for your feedback! Could you please give us some debug info? You can get the debug info with command:
```powershell
$Certificates = Get-AzWebAppCertificate -ResourceGroupName "mySampleResourceGroupName" -Debug
```

**Comment by RPCMarkR** [comment]
Update: We share the ASP with multiple clients. I initially upload the certificate from a client in another Resource Group, (the actual certificate is stored in the ASP). The error happens when I try to retrieve the certificate from the new client, in a different resource group. My script goes past the $certificates -ne $null portion and tries to upload the certificate again, which produces a conflict error.
Scenario: ASP = myASP, resourcegroup = myASPResourceGroup
Upload certificate for web app in myResourceGroup1, which uses myASP = success
**Note: Get-AzWebAppCertificate works properly at this point, for ResourceGroup1**
Check for certificate for web app #2 in myResourceGroup2, which uses myASP = nothing returned
Attempt to upload certificate for myResourceGroup2 = conflict error.
(I updated the script in my reproduce area above, with the additional code)

I believe the issue is because the certificate is stored in the ASP, not in the web app?

Here is the debug information you requested:
DEBUG: 5:30:34 PM - GetAzureWebAppCertificate begin processing with ParameterSet '__AllParameterSets'.

Confirm
Continue with this operation?
[Y] Yes  [A] Yes to All  [H] Halt Command  [S] Suspend  [?] Help (default is "Y"): a
DEBUG: 5:30:37 PM - using account id 'myemail@mydomain.com'...
DEBUG: [Common.Authentication]: Authenticating using Account: 'myemail@mydomain.com', environment: 'AzureCloud', tenant:
 '679af2bc-49ae-46da-a91c-xxxxxxxxxx'
DEBUG: [Common.Authentication]: Authenticating using configuration values: Domain:
'679af2bc-49ae-46da-a91c-xxxxxxxxxx', Endpoint: 'https://login.microsoftonline.com/', ClientId:
'1950a258-227b-4e31-a9cf-xxxxxxxxxx', ClientRedirect: 'urn:ietf:wg:oauth:2.0:oob', ResourceClientUri:
'https://management.core.windows.net/', ValidateAuthority: 'True'
DEBUG: [Common.Authentication]: Acquiring token using context with Authority
'https://login.microsoftonline.com/679af2bc-49ae-46da-a91c-ec8a45c212c6/', CorrelationId:
'00000000-0000-0000-0000-000000000000', ValidateAuthority: 'True'
DEBUG: [Common.Authentication]: Acquiring token using AdalConfiguration with Domain:
'679af2bc-49ae-46da-a91c-ec8a45c212c6', AdEndpoint: 'https://login.microsoftonline.com/', ClientId:
'1950a258-227b-4e31-a9cf-717495945fc2', ClientRedirectUri: urn:ietf:wg:oauth:2.0:oob
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0610653Z: ecbc08a4-3e8b-4ca7-acdc-852619a85673 - LoggerBase.cs: ADAL
PCL.Desktop with assembly version '3.19.2.6005', file version '3.19.50302.0130' and informational version
'2a8bec6c4c76d0c1ef819b55bdc3cda2d2605056' is running...
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0610653Z: ecbc08a4-3e8b-4ca7-acdc-852619a85673 - LoggerBase.cs: ADAL
PCL.Desktop with assembly version '3.19.2.6005', file version '3.19.50302.0130' and informational version
'2a8bec6c4c76d0c1ef819b55bdc3cda2d2605056' is running...
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0610653Z: ecbc08a4-3e8b-4ca7-acdc-852619a85673 - LoggerBase.cs: ===
Token Acquisition started:
 CacheType: null
 Authentication Target: User
 , Authority Host: login.microsoftonline.com
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0610653Z: ecbc08a4-3e8b-4ca7-acdc-852619a85673 - LoggerBase.cs: ===
Token Acquisition started:
 Authority: https://login.microsoftonline.com/679af2bc-49ae-46da-a91c-xxxxxxxxxx/
 Resource: https://management.core.windows.net/
 ClientId: 1950a258-227b-4e31-a9cf-717495945fc2
 CacheType: null
 Authentication Target: User

DEBUG: [ADAL]: Verbose: 2020-02-14T22:30:37.0640641Z: ecbc08a4-3e8b-4ca7-acdc-852619a85673 - LoggerBase.cs: Loading
from cache.
DEBUG: [ADAL]: Verbose: 2020-02-14T22:30:37.0640641Z: ecbc08a4-3e8b-4ca7-acdc-852619a85673 - LoggerBase.cs: Loading
from cache.
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0650670Z: 00000000-0000-0000-0000-000000000000 - LoggerBase.cs:
Deserialized 4 items to token cache.
DEBUG: [ADAL]: Verbose: 2020-02-14T22:30:37.0650670Z: ecbc08a4-3e8b-4ca7-acdc-852619a85673 - LoggerBase.cs: Looking up
cache for a token...
DEBUG: [ADAL]: Verbose: 2020-02-14T22:30:37.0650670Z: ecbc08a4-3e8b-4ca7-acdc-852619a85673 - LoggerBase.cs: Looking up
cache for a token...
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0650670Z: ecbc08a4-3e8b-4ca7-acdc-852619a85673 - LoggerBase.cs: An item
 matching the requested resource was found in the cache
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0650670Z: ecbc08a4-3e8b-4ca7-acdc-852619a85673 - LoggerBase.cs: An item
 matching the requested resource was found in the cache
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0650670Z: ecbc08a4-3e8b-4ca7-acdc-852619a85673 - LoggerBase.cs:
59.1206822166667 minutes left until token in cache expires
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0650670Z: ecbc08a4-3e8b-4ca7-acdc-852619a85673 - LoggerBase.cs:
59.1206822166667 minutes left until token in cache expires
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0650670Z: ecbc08a4-3e8b-4ca7-acdc-852619a85673 - LoggerBase.cs: A
matching item (access token or refresh token or both) was found in the cache
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0650670Z: ecbc08a4-3e8b-4ca7-acdc-852619a85673 - LoggerBase.cs: A
matching item (access token or refresh token or both) was found in the cache
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0650670Z: ecbc08a4-3e8b-4ca7-acdc-852619a85673 - LoggerBase.cs: ===
Token Acquisition finished successfully. An access token was returned: Expiration Time: 2/14/2020 11:29:44 PM +00:00
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0650670Z: ecbc08a4-3e8b-4ca7-acdc-852619a85673 - LoggerBase.cs: ===
Token Acquisition finished successfully. An access token was returned: Expiration Time: 2/14/2020 11:29:44 PM
+00:00Access Token Hash: KWfY4k5xVKEqDSKQbtjnb/xoHYD4ed39LRVy87fYeug=
  User id: e9b6416d-0582-4e95-8f7c-2153b764634b
DEBUG: [Common.Authentication]: Authenticating using Account: 'myemail@mydomain.com', environment: 'AzureCloud', tenant:
 '679af2bc-49ae-46da-a91c-xxxxxxxxxx'
DEBUG: [Common.Authentication]: Authenticating using configuration values: Domain:
'679af2bc-49ae-46da-a91c-xxxxxxxxxx', Endpoint: 'https://login.microsoftonline.com/', ClientId:
'1950a258-227b-4e31-a9cf-717495945fc2', ClientRedirect: 'urn:ietf:wg:oauth:2.0:oob', ResourceClientUri:
'https://management.core.windows.net/', ValidateAuthority: 'True'
DEBUG: [Common.Authentication]: Acquiring token using context with Authority
'https://login.microsoftonline.com/679af2bc-49ae-46da-a91c-xxxxxxxxxx/', CorrelationId:
'00000000-0000-0000-0000-000000000000', ValidateAuthority: 'True'
DEBUG: [Common.Authentication]: Acquiring token using AdalConfiguration with Domain:
'679af2bc-49ae-46da-a91c-xxxxxxxxxx', AdEndpoint: 'https://login.microsoftonline.com/', ClientId:
'1950a258-227b-4e31-a9cf-717495945fc2', ClientRedirectUri: urn:ietf:wg:oauth:2.0:oob
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0660642Z: 3c8c2717-4d44-4d9c-8d49-1c7da19b4d42 - LoggerBase.cs: ADAL
PCL.Desktop with assembly version '3.19.2.6005', file version '3.19.50302.0130' and informational version
'2a8bec6c4c76d0c1ef819b55bdc3cda2d2605056' is running...
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0660642Z: 3c8c2717-4d44-4d9c-8d49-1c7da19b4d42 - LoggerBase.cs: ADAL
PCL.Desktop with assembly version '3.19.2.6005', file version '3.19.50302.0130' and informational version
'2a8bec6c4c76d0c1ef819b55bdc3cda2d2605056' is running...
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0660642Z: 3c8c2717-4d44-4d9c-8d49-1c7da19b4d42 - LoggerBase.cs: ===
Token Acquisition started:
 CacheType: null
 Authentication Target: User
 , Authority Host: login.microsoftonline.com
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0660642Z: 3c8c2717-4d44-4d9c-8d49-1c7da19b4d42 - LoggerBase.cs: ===
Token Acquisition started:
 Authority: https://login.microsoftonline.com/679af2bc-49ae-46da-a91c-xxxxxxxxxx/
 Resource: https://management.core.windows.net/
 ClientId: 1950a258-227b-4e31-a9cf-717495945fc2
 CacheType: null
 Authentication Target: User

DEBUG: [ADAL]: Verbose: 2020-02-14T22:30:37.0660642Z: 3c8c2717-4d44-4d9c-8d49-1c7da19b4d42 - LoggerBase.cs: Loading
from cache.
DEBUG: [ADAL]: Verbose: 2020-02-14T22:30:37.0660642Z: 3c8c2717-4d44-4d9c-8d49-1c7da19b4d42 - LoggerBase.cs: Loading
from cache.
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0670639Z: 00000000-0000-0000-0000-000000000000 - LoggerBase.cs:
Deserialized 4 items to token cache.
DEBUG: [ADAL]: Verbose: 2020-02-14T22:30:37.0670639Z: 3c8c2717-4d44-4d9c-8d49-1c7da19b4d42 - LoggerBase.cs: Looking up
cache for a token...
DEBUG: [ADAL]: Verbose: 2020-02-14T22:30:37.0670639Z: 3c8c2717-4d44-4d9c-8d49-1c7da19b4d42 - LoggerBase.cs: Looking up
cache for a token...
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0670639Z: 3c8c2717-4d44-4d9c-8d49-1c7da19b4d42 - LoggerBase.cs: An item
 matching the requested resource was found in the cache
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0670639Z: 3c8c2717-4d44-4d9c-8d49-1c7da19b4d42 - LoggerBase.cs: An item
 matching the requested resource was found in the cache
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0670639Z: 3c8c2717-4d44-4d9c-8d49-1c7da19b4d42 - LoggerBase.cs:
59.120648935 minutes left until token in cache expires
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0670639Z: 3c8c2717-4d44-4d9c-8d49-1c7da19b4d42 - LoggerBase.cs:
59.120648935 minutes left until token in cache expires
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0670639Z: 3c8c2717-4d44-4d9c-8d49-1c7da19b4d42 - LoggerBase.cs: A
matching item (access token or refresh token or both) was found in the cache
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0670639Z: 3c8c2717-4d44-4d9c-8d49-1c7da19b4d42 - LoggerBase.cs: A
matching item (access token or refresh token or both) was found in the cache
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0670639Z: 3c8c2717-4d44-4d9c-8d49-1c7da19b4d42 - LoggerBase.cs: ===
Token Acquisition finished successfully. An access token was returned: Expiration Time: 2/14/2020 11:29:44 PM +00:00
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0680710Z: 3c8c2717-4d44-4d9c-8d49-1c7da19b4d42 - LoggerBase.cs: ===
Token Acquisition finished successfully. An access token was returned: Expiration Time: 2/14/2020 11:29:44 PM
+00:00Access Token Hash: KWfY4k5xVKEqDSKQbtjnb/xoHYD4ed39LRVy87fYeug=
  User id: e9b6416d-0582-4e95-8f7c-2153b764634b
DEBUG: [Common.Authentication]: Authenticating using Account: 'myemail@mydomain.com', environment: 'AzureCloud', tenant:
 '679af2bc-49ae-46da-a91c-xxxxxxxxxx'
DEBUG: [Common.Authentication]: Authenticating using configuration values: Domain:
'679af2bc-49ae-46da-a91c-xxxxxxxxxx', Endpoint: 'https://login.microsoftonline.com/', ClientId:
'1950a258-227b-4e31-a9cf-717495945fc2', ClientRedirect: 'urn:ietf:wg:oauth:2.0:oob', ResourceClientUri:
'https://management.core.windows.net/', ValidateAuthority: 'True'
DEBUG: [Common.Authentication]: Acquiring token using context with Authority
'https://login.microsoftonline.com/679af2bc-49ae-46da-a91c-xxxxxxxxxx/', CorrelationId:
'00000000-0000-0000-0000-000000000000', ValidateAuthority: 'True'
DEBUG: [Common.Authentication]: Acquiring token using AdalConfiguration with Domain:
'679af2bc-49ae-46da-a91c-xxxxxxxxxx', AdEndpoint: 'https://login.microsoftonline.com/', ClientId:
'1950a258-227b-4e31-a9cf-717495945fc2', ClientRedirectUri: urn:ietf:wg:oauth:2.0:oob
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0680710Z: 0f2b9149-c18f-4cc5-97f6-c6382cc610fd - LoggerBase.cs: ADAL
PCL.Desktop with assembly version '3.19.2.6005', file version '3.19.50302.0130' and informational version
'2a8bec6c4c76d0c1ef819b55bdc3cda2d2605056' is running...
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0680710Z: 0f2b9149-c18f-4cc5-97f6-c6382cc610fd - LoggerBase.cs: ADAL
PCL.Desktop with assembly version '3.19.2.6005', file version '3.19.50302.0130' and informational version
'2a8bec6c4c76d0c1ef819b55bdc3cda2d2605056' is running...
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0680710Z: 0f2b9149-c18f-4cc5-97f6-c6382cc610fd - LoggerBase.cs: ===
Token Acquisition started:
 CacheType: null
 Authentication Target: User
 , Authority Host: login.microsoftonline.com
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0680710Z: 0f2b9149-c18f-4cc5-97f6-c6382cc610fd - LoggerBase.cs: ===
Token Acquisition started:
 Authority: https://login.microsoftonline.com/679af2bc-49ae-46da-a91c-xxxxxxxxxx/
 Resource: https://management.core.windows.net/
 ClientId: 1950a258-227b-4e31-a9cf-717495945fc2
 CacheType: null
 Authentication Target: User

DEBUG: [ADAL]: Verbose: 2020-02-14T22:30:37.0690700Z: 0f2b9149-c18f-4cc5-97f6-c6382cc610fd - LoggerBase.cs: Loading
from cache.
DEBUG: [ADAL]: Verbose: 2020-02-14T22:30:37.0690700Z: 0f2b9149-c18f-4cc5-97f6-c6382cc610fd - LoggerBase.cs: Loading
from cache.
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0690700Z: 00000000-0000-0000-0000-000000000000 - LoggerBase.cs:
Deserialized 4 items to token cache.
DEBUG: [ADAL]: Verbose: 2020-02-14T22:30:37.0690700Z: 0f2b9149-c18f-4cc5-97f6-c6382cc610fd - LoggerBase.cs: Looking up
cache for a token...
DEBUG: [ADAL]: Verbose: 2020-02-14T22:30:37.0700668Z: 0f2b9149-c18f-4cc5-97f6-c6382cc610fd - LoggerBase.cs: Looking up
cache for a token...
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0700668Z: 0f2b9149-c18f-4cc5-97f6-c6382cc610fd - LoggerBase.cs: An item
 matching the requested resource was found in the cache
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0700668Z: 0f2b9149-c18f-4cc5-97f6-c6382cc610fd - LoggerBase.cs: An item
 matching the requested resource was found in the cache
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0700668Z: 0f2b9149-c18f-4cc5-97f6-c6382cc610fd - LoggerBase.cs:
59.1205988866667 minutes left until token in cache expires
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0700668Z: 0f2b9149-c18f-4cc5-97f6-c6382cc610fd - LoggerBase.cs:
59.1205988866667 minutes left until token in cache expires
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0700668Z: 0f2b9149-c18f-4cc5-97f6-c6382cc610fd - LoggerBase.cs: A
matching item (access token or refresh token or both) was found in the cache
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0700668Z: 0f2b9149-c18f-4cc5-97f6-c6382cc610fd - LoggerBase.cs: A
matching item (access token or refresh token or both) was found in the cache
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0700668Z: 0f2b9149-c18f-4cc5-97f6-c6382cc610fd - LoggerBase.cs: ===
Token Acquisition finished successfully. An access token was returned: Expiration Time: 2/14/2020 11:29:44 PM +00:00
DEBUG: [ADAL]: Information: 2020-02-14T22:30:37.0700668Z: 0f2b9149-c18f-4cc5-97f6-c6382cc610fd - LoggerBase.cs: ===
Token Acquisition finished successfully. An access token was returned: Expiration Time: 2/14/2020 11:29:44 PM
+00:00Access Token Hash: KWfY4k5xVKEqDSKQbtjnb/xoHYD4ed39LRVy87fYeug=
  User id: e9b6416d-0582-4e95-8f7c-2153b764634b
DEBUG: [Common.Authentication]: Renewing Token with Type: 'Bearer', Expiry: '02/14/2020 23:29:44 +00:00',
MultipleResource? 'True', Tenant: '679af2bc-49ae-46da-a91c-xxxxxxxxxx', UserId: 'myemail@mydomain.com'
DEBUG: [Common.Authentication]: User info for token DisplayId: 'myemail@mydomain.com', Name: Mark Renfrew, IdProvider:
'https://sts.windows.net/679af2bc-49ae-46da-a91c-xxxxxxxxxx/', Uid: 'e9b6416d-0582-4e95-8f7c-2153b764634b'
DEBUG: [Common.Authentication]: Checking token expiration, token expires '02/14/2020 23:29:44 +00:00' Comparing to
'02/14/2020 22:30:37 +00:00' With threshold '00:05:00', calculated time until token expiry: '00:59:07.2339360'
DEBUG: ============================ HTTP REQUEST ============================

HTTP Method:
GET

Absolute Uri:
https://management.azure.com/subscriptions/46a25c82-b36c-4269-a2e3-9f5bef8177e9/resourceGroups/CWH_rpcdisability/resour
ces?$filter=resourceType eq 'Microsoft.Web/Certificates'&api-version=2016-09-01

Headers:
x-ms-client-request-id        : 88658ea8-90a3-4160-9e92-e9b32eedcebe
accept-language               : en-US

Body:


DEBUG: ============================ HTTP RESPONSE ============================

Status Code:
OK

Headers:
Pragma                        : no-cache
x-ms-ratelimit-remaining-subscription-reads: 11999
x-ms-request-id               : aa04e2cb-ca00-48ea-8ad4-214d77d2ab8e
x-ms-correlation-request-id   : aa04e2cb-ca00-48ea-8ad4-214d77d2ab8e
x-ms-routing-request-id       : NORTHCENTRALUS:20200214T223114Z:aa04e2cb-ca00-48ea-8ad4-214d77d2ab8e
Strict-Transport-Security     : max-age=31536000; includeSubDomains
X-Content-Type-Options        : nosniff
Cache-Control                 : no-cache
Date                          : Fri, 14 Feb 2020 22:31:14 GMT

Body:
{
  "value": []
}

DEBUG: AzureQoSEvent: CommandName - Get-AzWebAppCertificate; IsSuccess - True; Duration - 00:00:02.6735043; Exception -
 ;
DEBUG: Finish sending metric.
DEBUG: 5:30:38 PM - GetAzureWebAppCertificate end processing.
DEBUG: 5:30:38 PM - GetAzureWebAppCertificate end processing.

**Comment by ThejaChoudary** [comment]
@RPCMarkR we are unable to repro the issue.Get-AzWebAppCertificate gets information about Azure Web App certificates associated with a specific resource group.And when we dont have any certificates associated with the webapps under the resource group it returns null.Please check if your webapp is having certificates associated with it .Let us know if your still facing the issue.
Thanks!

**Comment by SatishBoddu-MSFT** [comment]
Hello @Kotasudhakarreddy Could you please let us know if we are good to close this issue or any further action is needed?

**Comment by Kotasudhakarreddy** [comment]
closing due to lack of response. Please file a new issue if this is still a problem.
